packages/edit-site/src/components/sidebar-navigation-item/style.scss

```diff
@@ -1,6 +1,7 @@
 .edit-site-sidebar-navigation-item.components-item {
 	color: $gray-600;
 	border-width: $border-width-tab;
+	margin: 0 2.5px; // This combines with the border above to create a 4px gap and ensure the text line up nicely with other side bar elements along the Y axis
```

Comment:
Ack, this one's kinda ugly. It looks like this is to enable the focus style. I think that actually puts our metrics off in other places too, like the 40px item-height. If it's a nightmare to refactor the focus style to use a box-shadow + transparent outline (for high contrast mode) like our buttons do, I'm happy to approve this one. But that seems like a refactor to do eventually, what do you think?

Reply:
Seems the border wasn't actually required, so I removed it. A small margin is still required for the alignment though.


https://user-images.githubusercontent.com/846565/224068193-6558766b-44fd-40fe-aec9-4b6d338ccdf4.mp4

Reply:
Oh nice, you solved it, let me review in a second. This one:

> I'll see if I can move it to a pseudo element, but that feels a bit like replacing one hack with another xD

It should be possible to use box-shadow for the focus style, which wouldn't affect the footprint whereas border does.

Reply:
I just removed the border.

The focus style now comes from `.css-1xahv0o-View-unstyledButton-medium-item-spacedAround:focus` which seems to be connected to the experimental `ItemGroup` / `Item` components.

I agree it would be nice to update that to use `box-shadow` rather than the current `border` implementation. But since that's at the component level it should probably be handled in a dedicated PR?